src/types/steps.ts

```diff
@@ -47,6 +49,8 @@ export type StepOptionWelcome = {
   nextButton?: string
 }
 
+export type StepOptionAuth = { string: any }
```

Comment:
This should be more specific, what kinds of `string` value should be?

Reply:
Should be, just not sure about the granularity. Any ideas @kopijunkie @stefaniacardenas :thinking: